packages/edition-twig/dependencyGraph.json

```diff
@@ -0,0 +1 @@
:{"compileState":"clean"}},{"v":"01-molecules/01-layout/01-two-up.twig","value":{"compileState":"clean"}},{"v":"01-molecules/01-layout/00-one-up.twig","value":{"compileState":"clean"}},{"v":"01-molecules/01-layout/02-three-up.twig","value":{"compileState":"clean"}},{"v":"01-molecules/01-layout/03-four-up.twig","value":{"compileState":"clean"}},{"v":"01-molecules/02-blocks/00-media-block.twig","value":{"compileState":"clean"}},{"v":"01-molecules/02-blocks/01-block-headline-byline.twig","value":{"compileState":"clean"}},{"v":"01-molecules/02-blocks/02-block-hero.twig","value":{"compileState":"clean"}},{"v":"01-molecules/02-blocks/03-block-thumb-headline.twig","value":{"compileState":"clean"}},{"v":"01-molecules/02-blocks/04-block-headline.twig","value":{"compileState":"clean"}},{"v":"01-molecules/02-blocks/05-block-inset.twig","value":{"compileState":"clean"}},{"v":"01-molecules/03-media/00-figure-with-caption.twig","value":{"compileState":"clean"}},{"v":"01-molecules/04-forms/01-comment-form.twig","value":{"compileState":"clean"}},{"v":"01-molecules/04-forms/00-search.twig","value":{"compileState":"clean"}},{"v":"01-molecules/04-forms/02-newsletter.twig","value":{"compileState":"clean"}},{"v":"01-molecules/05-navigation/01-footer-nav.twig","value":{"compileState":"clean"}},{"v":"01-molecules/05-navigation/00-primary-nav.twig","value":{"compileState":"clean"}},{"v":"01-molecules/05-navigation/02-breadcrumbs.twig","value":{"compileState":"clean"}},{"v":"01-molecules/05-navigation/03-pagination.twig","value":{"compileState":"clean"}},{"v":"01-molecules/05-navigation/04-tabs.twig","value":{"compileState":"clean"}},{"v":"01-molecules/06-components/00-social-share.twig","value":{"compileState":"clean"}},{"v":"01-molecules/06-components/01-accordion.twig","value":{"compileState":"clean"}},{"v":"01-molecules/06-components/02-single-comment.twig","value":{"compileState":"clean"}},{"v":"01-molecules/07-messaging/00-alert.twig","value":{"compileState":"clean"}},{"v":"02-organisms/00-global/00-header.twig","value":{"compileState":"clean"}},{"v":"02-organisms/00-global/01-footer.twig","value":{"compileState":"clean"}},{"v":"02-organisms/01-article/00-article-body.twig","value":{"compileState":"clean"}},{"v":"02-organisms/02-comments/00-comment-thread.twig","value":{"compileState":"clean"}},{"v":"02-organisms/03-sections/00-latest-posts.twig","value":{"compileState":"clean"}},{"v":"02-organisms/03-sections/01-recent-tweets.twig","value":{"compileState":"clean"}},{"v":"02-organisms/03-sections/02-related-posts.twig","value":{"compileState":"clean"}},{"v":"03-templates/00-layouts/00-site.twig","value":{"compileState":"clean"}},{"v":"03-templates/00-layouts/01-page-1col.twig","value":{"compileState":"clean"}},{"v":"03-templates/00-layouts/01-page-2col.twig","value":{"compileState":"clean"}},{"v":"04-pages/00-homepage~emergency.json","value":{"compileState":"clean"}}],"edges":[{"v":"04-pages/00-homepage~emergency.json","w":"04-pages/00-homepage.twig","value":{}}]}}
```

Comment:
This file is an artifact of sorts and should not be initially available as part of generation. It should not be ignored either,

Reply:
I'm not following @bmuenzenmeyer 's logic. We work with usually 2 devs here doing Patternlab. 99% of the time we are editing patterns, assets, or corresponding json. If one of the devs pushes changes, we just pull and then start where they left off. I don't see how this autogenerated file would help.

My $0.02 -- If they push changes, this file would autogenerate regardless because I'd have to run PL regardless. I don't see a case for pushing this file so that someone could investigate this file and see the state that everything is in.

In fact, if someone really wanted to see this, they could edit their own .gitignore to allow it.

That's my $0.02, not sure if I'm helping or not here

Reply:
As the person who implemented that feature: 
This file caches the current build status of each pattern, so if either the data or the meta data changes after the last build, the file gets recompiled incrementally with all descendant patterns.

It can be regarded as a *local* build cache and thus should be added to .gitignore.

The alternative would be hell:
Version patterns, data and the generated output files with correct modification timestamps. This would lead to lots of potential merge conflicts in the dependency graph file when a few developers collaboratem, so
*don't do this, ever*.

Reply:
@hdwebpros 

> I don't see a case for pushing this file so that someone could investigate this file and see the state that everything is in.

Worth noting that no one would really look at this file manually. 

@tburny 

> It can be regarded as a local build cache and thus should be added to .gitignore.

thanks for chiming in! ignore it is then